internal/httpclient/README.md

```diff
@@ -91,6 +91,7 @@ Class | Method | HTTP request | Description
 *V0alpha1Api* | [**AdminCreateSelfServiceRecoveryLink**](docs/V0alpha1Api.md#admincreateselfservicerecoverylink) | **Post** /recovery/link | Create a Recovery Link
 *V0alpha1Api* | [**AdminDeleteIdentity**](docs/V0alpha1Api.md#admindeleteidentity) | **Delete** /identities/{id} | Delete an Identity
 *V0alpha1Api* | [**AdminGetIdentity**](docs/V0alpha1Api.md#admingetidentity) | **Get** /identities/{id} | Get an Identity
+*V0alpha1Api* | [**AdminIdentitySession**](docs/V0alpha1Api.md#adminidentitysession) | **Get** /sessions/identity/{id} | Calling this endpoint issues session for given identity.
```

Comment:
```suggestion
*V0alpha1Api* | [**AdminIdentitySession**](docs/V0alpha1Api.md#adminidentitysession) | **Get** /sessions/identity/{id} | Calling this endpoint issues a session for a given identity.
```

Reply:
Actually, since we're not proposing this upstream not sure if that's that big of an issue...

Reply:
Regenerated docs